csrank/objectranking/lambdamart.py

```diff
@@ -0,0 +1,370 @@
+import logging
+from collections import deque
+from multiprocessing import Pool
+
+import numpy as np
+from sklearn.tree import DecisionTreeRegressor
+
+from csrank.learner import Learner
+from csrank.metrics import ndcg_at_k
+from csrank.objectranking.object_ranker import ObjectRanker
+
+
+class LambdaMART(ObjectRanker,Learner):
+    def __init__(self, n_objects=None, n_object_features=None, number_of_trees=5, learning_rate=0.1,
+                 min_samples_split=2, max_depth=50, min_samples_leaf=1, max_leaf_nodes=None):
+        """
+            Create a LambdaMART based rank regression model. This model uses an ensemble of trees that learn to predict
```

Comment:
It would be useful to add a reference to the paper in the docstring.
See [here](https://github.com/kiudee/cs-ranking/blob/4b11e43171c609a7219969a3a305128ac276d766/csrank/objectranking/expected_rank_regression.py#L58) for how to do that.